packages/plugins/components/src/Utils/index.tsx

```diff
@@ -0,0 +1,7 @@
+export const makeRoute = (baseUrl: string, path: string) => `${baseUrl}${path}`;
+
+export const getBasePathName = () => {
+  const pathName = window.location.pathname;
+
+  return `/${pathName?.split('/')?.[1]}`;
+};
```

Comment:
nit:
should this folder's name be capitalized (`Utils`)? it's not gonna be a component, and seems out of pattern with other non-component folders, ie `routes`, touched in this PR